### PR TITLE
Fix broken CI tests and test release code

### PR DESCRIPTION
**Pull request by rakeshkashyap123**
1. Test release branch
2. Exclude avroutil package from feathr
3. Fix broken tests

### CODE REVIEW

**Comment by xiaoyongzhu** [comment]
Let's get this in, seems we only have one failing test case and let's fix them in the next PR.

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
-version=0.11.1-rc1
+version=v0.11.1-rc1
```
Comment (blrchen) [inline gradle.properties:1]: Please use v0.10.3-rc5 as next publis release is v0.10.

**Comment by blrchen** [comment]
Thanks @rakeshkashyap123 for fixing CI issues. Suggest update PR title to mention what this PR is for.

I think one of remaining works is update [publish-to-maven.yml](https://github.com/feathr-ai/feathr/blob/main/.github/workflows/publish-to-maven.yml) to publish 4 newly added jar files.

File: gradle.properties (hunk 1 above)
Comment (rakeshkashyap123) [reply gradle.properties]: Updated.

**Review by blrchen — DISMISSED** [review]

**Review by blrchen — APPROVED** [review]

**Review by xiaoyongzhu — APPROVED** [review]

**Pull request merged**